sql/catalyst/src/main/scala/org/apache/spark/sql/internal/StaticSQLConf.scala

```diff
@@ -126,4 +126,13 @@ object StaticSQLConf {
       .intConf
       .createWithDefault(1000)
 
+  val MAX_BROADCAST_EXCHANGE_THREADNUMBER =
+    buildStaticConf("spark.sql.broadcastExchange.maxThreadNumber")
+      .doc("The maximum degree of parallelism to fetch and broadcast the table.If we " +
```

Comment:
extra sapce -> `table. If`

Reply:
Sorry for this style @HyukjinKwon actually i found some other code has the same problem,can i open pr to fix that?

Reply:
If that's around this code and that's the only the one, yea, let's do that. If there are multiple across this files, let's don't include.